src/calculator.m

```diff
@@ -0,0 +1,22 @@
+classdef Calculator
+    methods(Static)
+        function result = add(a, b)
+            result = a + b;
+        end
+        
+        function result = subtract(a, b)
+            result = a - b;
+        end
+        
+        function result = multiply(a, b)
+            result = a * b;
+        end
+        
+        function result = divide(a, b)
+            if b == 0
+                error('Division by zero is undefined.');
+            end
```

Comment:
In the divide method, consider handling cases where 'a' is not numeric, in addition to checking 'b'.